components/camera/videosource/webcam.go

```diff
@@ -323,54 +318,55 @@ func (c *webcam) reconnectCamera(conf *WebcamConfig) error {
 // and once the resource is closed, so should the monitor. That is, it should
 // no longer send any resets once a Close on its associated resource has returned.
 func (c *webcam) Monitor() {
-	const wait = 500 * time.Millisecond
-	c.activeBackgroundWorkers.Add(1)
+	// const wait = 500 * time.Millisecond
```

Comment:
Instead of commenting this out, I'd just remove it entirely. Commented-out code tends to rot and not be usable in the future.